### ISSUE TITLE
Icons: Remove aria-hidden to fix narrator focus bug for Modal dialog and Panel

### ISSUE THREAD

**Issue by lavanya-kuda** [OfficeDev/office-ui-fabric-react#9808]
## Codepen demonstrating the issue

Remove aria-hidden to fix narrator focus bug for Modal dialog and Panel issue has been fixed in office-ui-fabric-react v6.180.0 but in our project we are using 6.195.0 which doesn't have the aria-hidden fix and still shows an accessibility issue. Does v6.195.0 have aria-hidden issue?

![image](https://user-images.githubusercontent.com/17567591/61206657-bd5fd380-a710-11e9-9b68-bc92c783b46f.png)

**Comment by micahgodbolt** [comment]
FYI here is the 6.180.0 release. Is it one of these changes? https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.180.0

**Comment by joschect** [comment]
@lavanya-kuda Could you be a bit more specific as to where you're seeing the problem? Is it in fabric document cards? Or just everywhere? Is there a codepen that repros it?

**Comment by lavanya-kuda** [comment]
Please refer to this PR (https://github.com/OfficeDev/office-ui-fabric-react/pull/9019) which fixed the issue on office-fabric-ui version v6.180.0. I don't see those are working in v6.195.0?

**Comment by joschect** [comment]
@lavanya-kuda I see that fix, but I don't know if that's impacting the scenario you're talking about, could you provide a codepen or something where you see the issue?

**Comment by joschect** [comment]
@lavanya-kuda more specifically, could you confirm which component/where you see the problem?

**Comment by lavanya-kuda** [comment]
This issue occurs for Panel , Modal component in v6.195.0 of office-ui-fabric-react. https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI Please see this error which  we have for Accessibility.

**Comment by joschect** [comment]
@lavanya-kuda I made this [codepen](https://codepen.io/slimemold/pen/eqWBjQ?&editable=true) could you confirm that the issue shows up here? I took a look and don't see any focusable aria-hiddens

**Comment by lavanya-kuda** [comment]
HI @joschect, I can't see the aria-hidden issue in this codepen. I ran accessibility tool. Which version of office fabric ui are you using?

**Comment by joschect** [comment]
@lavanya-kuda Per the unpkg I'm using the latest version of fabric 6 which is 6.197

![image](https://user-images.githubusercontent.com/20801176/62308227-e3d47d00-b439-11e9-95a2-c86d8aa11937.png)

**Comment by lavanya-kuda** [comment]
Thank you @joschect, we were in 6.195 and unable to see the fix. I have upgraded to 6.197 then I don't see the aria-hidden issue.

**Comment by dzearing** [comment]
It sounds like this issue has been resolved. @lavanya-kuda can you please close the issue once you've confirmed? Or, provide additional information on anything that you still feel is pending.
